config/config.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"gopkg.in/yaml.v3"
+)
+
+type Config struct {
+	Port          int    `yaml:"port"`
+	Adress_server string `yaml:"address_server"`
+	Hostname      string `yaml:"hostname"`
+	Start_html    string `yaml:"start-html"`
+	Web_directory string `yaml:"web-directory"`
+}
+
+func (c *Config) Parse(content []byte) error {
+	return yaml.Unmarshal(content, c)
+}
+
+func main() {
+	name_config := "config.yml"
+	fmt.Println("Wczytywanie plik konfiguracyjnego: ", name_config)
+	content, err := ioutil.ReadFile(name_config)
+	if err != nil {
+		fmt.Println("Nie udalo sie odczytac zawartosc pliku ", name_config)
+	}
+	var config Config
+	if err := config.Parse(content); err != nil {
+		fmt.Println(err)
```

Comment:
Tutaj też - będzie błąd, to tylko informujesz o nim, ale usiłujesz procesować dalej. Znowu warto rozważyć, czy nie zastopować.

Pewne struktury danych mogą Ci wywalić program jak spróbujesz je użyć bez upewnienia się, czy one zawierają jakieś dane. Tutaj `config` jest structem, więc do takich właśnie należy.